RELEASE.md

```diff
@@ -37,19 +37,44 @@ Below is what a typical release week may look like:
 
 ## Test release before it is rolling out.
 
-The release is a pre-release, it means it is not yet the latest version, so no clients will automatically update to this version.
+The release is a **pre-release**, it means it is not yet the latest version, so no clients will automatically update to this version.
 
-It allows QE (and everyone else) to test these binaries before they go live or are in the package managers.
+Inform QE the **pre-release** is out! It allows QE (and everyone else) to downnload and test these binaries before they go live or are in the package managers.
 
-At this step, generates as well Flathub pull request so people can test the package from the builders of the pull request.
+At this step, it also generates a Flathub pull request so people can test the package from the builders of the pull request.
```

Comment:
not sure about the 'it' changes in 'it also generates' as it's a person that needs to do that for now

Reply:
we do it manually now